src/main/java/ca/gc/aafc/agent/api/repository/PersonRepository.java

```diff
@@ -22,10 +23,12 @@
   public PersonRepository(
     @NonNull DinaService<Person> dinaService,
     @NonNull DinaFilterResolver filterResolver,
-    Optional<DinaAuthenticatedUser> authenticatedUser
+    Optional<DinaAuthenticatedUser> authenticatedUser,
+    Optional<DinaAuthorizationService> authorizationService
   ) {
     super(
       dinaService,
+      authorizationService,
```

Comment:
I think we should give it an Optional wrapping a null here as we haven't developed an authorization service for the agent module yet, We should be making one around dina roles soon